docs/intro.rst

```diff
@@ -1,8 +1,165 @@
+
+The CumulusCI suite of tools is part of the not-so-secret sauce that makes it possible to build and release products. CumulusCI automation runs throughout our development lifecycle, starting from feature branches through the delivery of the latest release. 
+
+* GitHub repositories serve as version-controlled source repositories.
+* The CumulusCI command-line interface, ``cci``, runs single-action tasks and multiple-action flows for development and testing.
+* MetaCI uses CumulusCI flows to build Salesforce managed packages from GitHub repositories.
+* MetaDeploy automates setup and configuration of customer orgs.
+
+You can use the very same automation used internally by Salesforce to quickly:
+
+* Build sophisticated orgs with dependencies automatically installed.
+* Load and capture sample datasets to make your orgs feel real.
+* Apply transformations to existing metadata to tailor orgs to your specific requirements.
+* Run builds in continuous integration systems.
+* Create end-to-end browser tests and set up automation using `Robot Framework <https://robotframework.org/>`_.
+
+The automation defined using CumulusCI is portable. It's stored in a source repository and can be run from your local command line, from a continuous integration system, or from a customer-facing installer. CumulusCI can run automation on scratch orgs created using the Salesforce CLI, or on persistent orgs like sandboxes, production orgs, and Developer Edition orgs.
+
+Finally, by way of introduction, CumulusCI is more than just a set of tools. It represents our holistic approach to product development. Rather than focusing on just the `Org Development Model <https://trailhead.salesforce.com/en/content/learn/modules/org-development-model>`_ or the `Package Development Model <https://trailhead.salesforce.com/en/content/learn/modules/sfdx_dev_model>`_,  Salesforce.org has implemented its own *Product Delivery Model* using CumulusCI. 
+
+
+
+The Product Delivery Model
+--------------------------
+
+The Product Delivery Model focuses on the customer experience, not on the technical artifacts you're delivering. When building a product, there are detailed technical considerations for whether an individual component is best distributed within a package, or as additional unpackaged metadata, or as setup automation that runs before or after a package is installed. CumulusCI makes it possible to deliver a first class, fully configured product that makes use of these different techniques. 
+
+CumulusCI automation, which makes it easy to create products that span multiple package repositories and include complex setup operations, best implements the Product Delivery Model, along with MetaDeploy and other applications in CumulusCI.
```

Comment:
The entire PDM subsection irks me. It feels like we're just coming up a bit short on the what and the how of PDM.  How exactly does it focus on the customer experience more than the technical artifact? What exactly *is* technical artifact (and if it's notable to point out here, why is this the only time in the entire manual that we use the phrase)? 

Maybe I'm just being nitpicky, but I feel like there's a sentence or two more to add here about what makes PDM such an important model for CCI and any devs working with it.

Reply:
I thought about this a little more. One thing that _may_ be worth adding would be a sentence that outlines a less-than-ideal delivery experience for a customer of a package that isn't using CumulusCI PDM vs. one that does. We could say something like, "it is not uncommon for paying customers of managed package products to be handed configuration steps that can take hours (or even days) to complete after initial installation of the product. Our product delivery model allows our teams to develop configurations directly into automated workflows, which in turn are used as part of the install process by customers which help deliver a usable out-of-the-box product."